CHANGELOG.rst

```diff
@@ -46,6 +46,9 @@ Changelog
   :meth:`~cryptography.x509.CertificateSigningRequest.get_attribute_for_oid`.
 * Added support for encoding attributes in certificate signing requests via
   :meth:`~cryptography.x509.CertificateSigningRequestBuilder.add_attribute`.
+* On OpenSSL 1.1.1d and higher ``cryptography`` now uses the built-in CSPRNG
+  instead of the OS random engine because these versions now properly reseed
+  on fork.
```

Comment:
```suggestion
* On OpenSSL 1.1.1d and higher ``cryptography`` now uses OpenSSL's built-in CSPRNG
  instead of it's own OS random engine because these versions of OpenSSL properly reseed
  on fork.
```

Reply:
Wrong its